examples/embedded/foodie-inc-2022-05-01.sssom.tsv

```diff
@@ -1,25 +1,30 @@
+# comment: We could map to FOODON:00004187 instead which more specifically refers to
+#   'raw' Pink apples. Decided against to be consistent with other mapping choices.
 # curie_map:
 #   FOODON: http://purl.obolibrary.org/obo/FOODON_
 #   KF_FOOD: https://kewl-foodie.inc/food/
+#   orcid: https://orcid.org/
+#   owl: http://www.w3.org/2002/07/owl#
+#   rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+#   rdfs: http://www.w3.org/2000/01/rdf-schema#
+#   semapv: https://w3id.org/semapv/
```

Comment:
Wait… Is the URL for `semapv` `https://w3id.org/semapv/` or `https://w3id.org/semapv/vocab/`???

All SEMAPV IDs in [the SEMAPV repository](https://github.com/mapping-commons/semantic-mapping-vocabulary/blob/main/semapv.owl) are of the form `https://w3id.org/semapv/vocab/XXX` (e.g. `https://w3id.org/semapv/vocab/LexicalMatching`).

Reply:
That is a big bug! Thanks for noticing. https://github.com/mapping-commons/sssom/pull/287

The model level prefix map overwrote the bioregistry context!

Reply:
Also fixed now in this PR